src/main/java/com/epam/ta/reportportal/core/organization/PersonalOrganizationService.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.ta.reportportal.core.organization;
+
+import com.epam.reportportal.api.model.OrganizationInfo;
+import com.epam.ta.reportportal.core.plugin.Pf4jPluginBox;
+import com.epam.ta.reportportal.entity.user.User;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+/**
+ * @author <a href="mailto:reingold_shekhtel@epam.com">Reingold Shekhtel</a>
+ */
+@Slf4j
+@Service
+public class PersonalOrganizationService {
+
+  private final Pf4jPluginBox pluginBox;
+
+  public PersonalOrganizationService(Pf4jPluginBox pluginBox) {
+    this.pluginBox = pluginBox;
+  }
+
+  public Optional<OrganizationInfo> create(User user) {
+    try {
+      return getOrgExtension().map(ext -> ext.createPersonalOrganization(user));
+    } catch (Exception e) {
+      log.info("Can't create personal organization for user: {}, reason: {}", user.getLogin(), e.getMessage());
```

Comment:
`log.error(...)` method is more applicable for the catch blocks

Reply:
No, because it's not error. This is business logic flow.